srcpkgs/libgccjit/template

```diff
@@ -0,0 +1,241 @@
+		ppc64*) _args+=" --with-abi=elfv2 --enable-secureplt --enable-targets=powerpc-linux";;
+		ppc*) _args+=" --enable-secureplt";;
+	esac
+
+	# fix: unknown long double size, cannot define BFP_FMT
+	case "$XBPS_TARGET_MACHINE" in
+		ppc*-musl) _args+=" --disable-decimal-float";;
+	esac
+
+
+	if [ "$CROSS_BUILD" ]; then
+		export CC_FOR_TARGET="$CC"
+		export GCC_FOR_TARGET="$CC"
+		export CXX_FOR_TARGET="$CXX"
+
+		_args+=" --host=$XBPS_CROSS_TRIPLET"
+		_args+=" --with-build-sysroot=${XBPS_CROSS_BASE}"
+	elif [ -z "$CHROOT_READY" ]; then
+		export LD_LIBRARY_PATH="${XBPS_MASTERDIR}/usr/lib${XBPS_TARGET_WORDSIZE}"
+		_args+=" --build=${_triplet}"
+	else
+		_langs="jit,lto"
+		_args+=" --build=${_triplet}"
+	fi
+
+	if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
+		_args+=" --disable-gnu-unique-object"
+		_args+=" --disable-symvers"
+		_args+=" libat_cv_have_ifunc=no"
+	else
+		_args+=" --enable-gnu-unique-object"
+	fi
+
+	case "$XBPS_TARGET_MACHINE" in
+		ppc*) _args+=" --disable-vtable-verify";;
+		*) _args+=" --enable-vtable-verify";;
+	esac
+
+	export CFLAGS="${CFLAGS/-D_FORTIFY_SOURCE=2/}"
+	export CXXFLAGS="${CXXFLAGS/-D_FORTIFY_SOURCE=2/}"
+
+	# Disable explicit -fno-PIE, gcc will figure this out itself.
+	export CFLAGS="${CFLAGS//-fno-PIE/}"
+	export CXXFLAGS="${CXXFLAGS//-fno-PIE/}"
+	export LDFLAGS="${LDFLAGS//-no-pie/}"
+
+	_args+=" --prefix=/usr"
+	_args+=" --mandir=/usr/share/man"
+	_args+=" --infodir=/usr/share/info"
+    _args+=" --with-bugurl='https://github.com/jashandeep-sohi/libgccjit-pkg/issues'" \
+	_args+=" --libexecdir=/usr/lib${XBPS_TARGET_WORDSIZE}"
+	_args+=" --libdir=/usr/lib${XBPS_TARGET_WORDSIZE}"
+	_args+=" --with-system-zlib"
+	_args+=" --with-linker-hash-style=$_hash"
+	_args+=" --enable-linker-build-id"
+	_args+=" --enable-threads=posix"
+	_args+=" --enable-lto"
+	_args+=" --enable-shared"
+    _args+=" --enable-host-shared"
+	_args+=" --enable-checking=release"
+	_args+=" --enable-plugins"
+	_args+=" --enable-default-ssp"
+	_args+=" --enable-default-pie"
+	_args+=" --disable-multilib"
+	_args+=" --disable-bootstrap"
+	_args+=" --enable-__cxa_atexit"
+    _args+=" --disable-libquadmath"
+    _args+=" --disable-libquadmath-support"
+    _args+=" --disable-libgomp" 
+    _args+=" --disable-liboffloadmic" 
+    _args+=" --disable-libada"
+    _args+=" --disable-libvtv"
+    _args+=" --disable-libsanitizer"
+	_args+=" --disable-werror"
+	_args+=" --disable-nls"
+	_args+=" --disable-libstdcxx-pch"
+	_args+=" --disable-sjlj-exceptions"
+	_args+=" --disable-target-libiberty"
+
+	mkdir -p build
+	cd build
+
+    # If we are checking packages, we need to have c++ compiled
+    [ -z "$XBPS_CHECK_PKGS" ] && _langs="jit" || _langs="jit,c++"
+    _args+=" --enable-languages=${_langs}"
+    CONFIG_SHELL=/bin/bash \
+	    ${wrksrc}/configure ${_args}
+
+}
+
+do_build() {
+	if [ -z "$CHROOT_READY" ]; then
+		export LD_LIBRARY_PATH="${XBPS_MASTERDIR}/usr/lib${XBPS_TARGET_WORDSIZE}"
+	fi
```

Comment:
I'm not sure this is at all necessary>

Reply:
the latest commit only commented out this section -_-'. I can remove these lines in a final polished commit